admin-dev/themes/new-theme/js/components/form/delta-quantity-input.ts

```diff
@@ -37,22 +36,14 @@ export type DeltaQuantityConfig = {
   modifiedQuantityClass: string;
   newQuantitySelector: string;
   initialQuantityPreviewSelector: string;
-  submittableDeltaConfig: SubmittableDeltaConfig|null;
 }
 
-export type SubmittableDeltaConfig = {
-  wrapperSelector: string;
-  submitCallback: (input: Element) => any;
-}
-
-export type InputDeltaQuantityConfig = Partial<DeltaQuantityConfig>;
-
-class DeltaQuantityInput {
+export default class DeltaQuantityInput {
   private config: DeltaQuantityConfig;
 
   private eventEmitter: EventEmitter;
 
-  constructor(config: InputDeltaQuantityConfig = {}) {
+  constructor(config: Partial<DeltaQuantityConfig> = {}) {
```

Comment:
Oh, nice one!! I'll have to copy this for other components, It prevents creating a useless additional type, thanks!